packages/instant/webpack.config.js

```diff
@@ -89,11 +89,17 @@ const generateConfig = (dischargeTarget, heapConfigOptions, rollbarConfigOptions
         });
     }
 
+    const infuraProjectId =
+        dischargeTarget === 'production'
+            ? process.env.INSTANT_INFURA_PROJECT_ID_PRODUCTION
+            : process.env.INSTANT_INFURA_PROJECT_ID_DEVELOPMENT;
+
     const envVars = {
         GIT_SHA: JSON.stringify(GIT_SHA),
         NPM_PACKAGE_VERSION: JSON.stringify(process.env.npm_package_version),
         ROLLBAR_ENABLED: rollbarEnabled,
-        HEAP_ENABLED: heapEnabled
+        HEAP_ENABLED: heapEnabled,
+        INSTANT_INFURA_PROJECT_ID: JSON.stringify(infuraProjectId),
```

Comment:
is `infuraProjectId` a string? if so, why do we want to serialize it to JSON?

Reply:
`infuraProjectId` is a number. This stringify call is effectively converting from a number to a string. This seems to be the convention we're following in this config file. I'm happy to change it up but we should prob be consistent either way. @fragosti thoughts?

Reply:
This is a fun little `DefinePlugin` gotcha
https://stackoverflow.com/questions/50454341/why-json-stringifyproduction